packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
+        aspectRatio:
+          type: string
+          description: Aspect ratio of the asset
+          example: 16:9
+        meta:
+          type: object
+          description: User input metadata associated with the asset
+          example:
+            {
+              "title": "My awesome video",
+              "description": "This is a video of my awesome life",
+              "tags": ["awesome", "life", "video"],
+            }
+        updatedAt:
+          type: string
+          description:
+            Timestamp (in milliseconds) at which the asset was last updated
+          example: 2022-01-01T02:20:00Z
+        createdAt:
+          type: string
+          description: Timestamp (in milliseconds) at which asset was created
+          example: 2022-01-01T02:20:00Z
+        format:
+          type: string
+          description: Format of the asset
+          example: mp4
+        hash:
+          type: string
+          description: Hash of the asset
+          example: sha256:hu31498hf14398
+        tracks:
+          type: array
+          description:
+            List of tracks associated with the asset when the format contemplate
+            them (e.g. mp4)
+          items:
+            type: object
+            properties:
+              kind:
+                type: string
+                description: Kind of track
+                example: video
+              width:
+                type: number
+                description: Width of the track - only for video tracks
+                example: 1920
+              height:
+                type: number
+                description: Height of the track - only for video tracks
+                example: 1080
+              fps:
+                type: number
+                description: Frame rate of the track - only for video tracks
+                example: 30
+              id:
+                type: string
+                description: ID of the track
+                example: ABC123
+              duration:
+                type: number
+                description: Duration of the track in seconds
+                example: 23.8238
+              bitrate:
+                type: number
+                description: Bitrate of the track in bits per second
+                example: 1000000
+              channels:
+                type: number
+                description: Amount of audio channels in the track
+                example: 2
+              sampleRate:
+                type: number
+                description:
+                  Sample rate of the track in samples per second - only for
+                  audio tracks
+                example: 44100
+              bitDepth:
+                type: number
+                description: Bit depth of the track - only for audio tracks
+                example: 16
+              codec:
+                type: string
+                description: Codec of the track
+                example: aac
```

Comment:
Is this only for audio as well?

Since the are so different in the specs, should we have separate `audioTracks` and `videoTracks` perhaps? Or would that be counter intuitive considering how other tools and platforms represent this?

Reply:
Yeah as I mentioned in another comment, a track can be either a audio or a video track. For example, this command:
```
ffprobe -v quiet -print_format json -show_format -show_streams example.mp4
```
would output a similar json, with one or more separate streams (`tracks`), either audio or video. In this case all the params are even the same for both the tracks.  
In an example case, `codec` could be `aac` or `h264` depending on which track it is

Reply:
Hmm I see. Yeah so let's follow the standard from `ffmpeg` then. I guess it could be useful for "newcomers" to have tracks explicitly separated about what are the audio and video tracks, but maybe it would feel alien for more experienced video-developers 😆